donut/modules/calendar/static/style.css

```diff
@@ -0,0 +1,144 @@
+body{
+    padding-bottom:0;
+    padding-top:0;
+}
+
+.checkbox-inline input[type=checkbox]{
+    margin-left: 0;
+}
+
+#cal td{
+    height: 30%;
+    overflow: scroll;
```

Comment:
Let's change all these to `overflow: auto` so the scroll bar isn't shown if it's not needed